docs/redis.rst

```diff
@@ -209,6 +208,12 @@ described in Heroku's own documentation.
           heroku config:unset REDIS_REQUIRED -a "$app"
       done
 
+12. After a period of stability, the old instance can be safely deleted:
+
+   .. code-block:: bash
+
+      heroku addons:destroy "$old_instance"
```

Comment:
_non-blocking_

The "period of stability" evaluation should be kept short, as the old instance will quickly become unsuitable/too-stale for rolling back to since the new instance will have potentially-significantly diverged from it. This is why I'd destroyed it ~immediately after successful upgrade in the past; it goes stale relatively quickly.

Reply:
Good point. Dropped 37280f60a55bc41a5ea01d33e756470c110a1762.

A better change would be to update the testing step

https://github.com/nextstrain/nextstrain.org/blob/18507ca25476372c4e960ba088e6e1b12f0dcc68/docs/redis.rst#L221

which only tests that replication was successful and not that new changes can be made. Such steps could be logout/login cycle, group membership changes, etc.